app/views/course/assessment/submission/submissions/_statistics.html.slim

```diff
@@ -44,4 +53,9 @@ div.panel.panel-default
           - @submission.answers.each do |answer|
             tr
               th = format_inline_text(answer.question.title)
-              td = answer.grade
+              td
+                span.submission-grades-summary-grade> (
+                    id="submission-grades-summary-answer-#{answer.id}-grade")
```

Comment:
The main content block for the `answer`s should have been rendered further up in the page already. Those are tagged with `content_tag_for`. Didn't put here also because we are just extracting the grade for the summary, not trying to render some representation of the object.